### PR TITLE
Feature/1 take over initial version from virtual schemas

### LINKED ISSUE #1: Take over initial version from Virtual Schemas
## Situation

[Virtual Schemas](https://github.com/exasol/virtual-schemas) version 7.3.0 and some below contain a test utility for capturing log messages. We move this one here as starting point for the project.

## Acceptance Criteria

1. `CapturingLogHandler` imported into this repository.
2. Documented in `README.md`

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #1.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+### Run Time Dependencies
+
+Running the Virtual Schema requires a Java Runtime version 9 or later.
+
+### Build Time Dependencies
+
+| Dependency                                                                          | Purpose                                                | License                       |
+|-------------------------------------------------------------------------------------|--------------------------------------------------------|-------------------------------|
+| [Java Hamcrest](http://hamcrest.org/JavaHamcrest/)                                  | Checking for conditions in code via matchers           | BSD License                   |
+| [J5SE](https://github.com/itsallcode/junit5-system-extensions)                      | JUnit5 extensions to test Java System.x functions      | Eclipse Public License 2.0    |
```
Comment (AnastasiiaSergienko) [inline README.md]: I think we don't use this dependency in the project. Do we need it for something?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.exasol</groupId>
+  <artifactId>java-util-logging-testing</artifactId>
+  <version>${product.version}</version>
+  <name>Test utilities for `java.util.logging`</name>
+  <description>This project provides utilities that help testing software that uses `java.util.logging` as its logging framework.</description>
+  <url>https://github.com/exasol/java-util-logging-testing</url>
+  <properties>
+    <product.version>2.2.0</product.version>
```
Comment (AnastasiiaSergienko) [inline pom.xml]: What will be a version of the project?

File: README.md (hunk 1 above)
Comment (redcatbear) [reply README.md]: Right. I removed the wrong dependency. Should be JUnit5 instead of J5SE.

File: pom.xml (hunk 2 above)
Comment (redcatbear) [reply pom.xml]: 1.0.0

**Review by AnastasiiaSergienko — DISMISSED** [review]

**Review by AnastasiiaSergienko — DISMISSED** [review]

**Review by AnastasiiaSergienko — APPROVED** [review]

**Pull request merged**